src/main/java/core/basesyntax/service/FileServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
+public class FileServiceImpl implements FileService {
+    private static final String MASSAGE = "Can't get data from file ";

```

Comment:
```suggestion
    private static final String MESSAGE = "Can't get data from file ";
```
fix typo in all similar places